package.json

```diff
@@ -154,6 +154,7 @@
         "setup": "yarn install && yarn clean && yarn build:i18n",
         "start": "yarn setup && yarn build:dev:dist --watch",
         "start:dev": "yarn setup && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev yarn webpack-dev-server --config scripts/webpack.config.js --mode development",
+        "start:npm": "f() { yarn link && yarn setup && IS_LINKED=1 yarn build:dev:dist --watch; }; f",
```

Comment:
I think people may prefer to link/unlink on themselves. So maybe `yarn link` is not necessary?

Reply:
Should this be `start:linked` to be consistent with other projects?